private-templates-service/client/package.json

```diff
@@ -39,7 +40,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "sh -ac '. ./.env.${REACT_APP_ENV}; react-scripts build'",
+    "build": "sh -ac 'react-scripts build'",
```

Comment:
What is the sh -ac doing? I removed it in my PR

Reply:
I'll remove it!